python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py

```diff
@@ -0,0 +1,153 @@
+import json
+import os
+import subprocess
+import textwrap
+from typing import Any, Callable, List, Mapping, Optional
+
+from dagster import OpDefinition, Output, SolidExecutionContext, check
+from dagster.core.asset_defs import AssetIn, asset
+
+
+def _load_manifest_for_project(
+    project_dir: str, profiles_dir: str, select: str
+) -> Mapping[str, Any]:
+    command_list = [
+        "dbt",
+        "ls",
+        "--project-dir",
+        project_dir,
+        "--profiles-dir",
+        profiles_dir,
+        "--select",
+        select,
+        "--resource-type",
+        "model",
+    ]
+    # running "dbt ls" regenerates the manifest.json, which includes a superset of the actual
+    # "dbt ls" output
+    subprocess.Popen(command_list, stdout=subprocess.PIPE).wait()
+    manifest_path = os.path.join(project_dir, "target", "manifest.json")
```

Comment:
the target directory is configurable by the user, so this path won't always be correct: https://docs.getdbt.com/reference/project-configs/target-path. You can probably just add a parameter for this.